server/routes/api/place.py

```diff
@@ -15,9 +15,10 @@
 import collections
 import json
 
-from flask import Blueprint, request
-
+from flask import Blueprint, request, jsonify
 from cache import cache
+import services.datacommons as dc
+# Note that significant code broke when this was changed to dc.fetch_data.
```

Comment:
This is worrisome.  I have no mental model for why this would happen.  They should be equivalent.  May we change the note to a todo to investigate?